.idea/fileTemplates/MolgenisEntityType.java

```diff
@@ -66,6 +66,7 @@ public void init() {
   }
 }
 
+@SuppressWarnings("unused")
```

Comment:
I can't get IntelliJ to use the updated template. Did you experience the same issue? Restarting IntelliJ and re-executing the file template setup (https://molgenis.gitbooks.io/molgenis/content/guide-using-an-ide-for-backend.html --> Use MOLGENIS file-templates) doesn't help.